tools/distro-scraper/scraper/models.py

```diff
@@ -0,0 +1,48 @@
+from pydantic import BaseModel, Field, field_validator
+
+
+class ImageItem(BaseModel):
```

Comment:
Should this forbid extra values too?

```suggestion
class ImageItem(BaseModel, extra="forbid"):
```